com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -99,6 +99,11 @@ internal void __endSendServerRpc(NetworkSerializer serializer, uint rpcMethodId,
             {
                 rpcQueueContainer.EndAddQueueItemToFrame(serializer.Writer, RpcQueueHistoryFrame.QueueFrameType.Outbound, NetworkUpdateStage.PostLateUpdate);
             }
+
+            if (NetworkManager.__rpc_name_table.TryGetValue(rpcMethodId, out var rpcMethodName))
+            {
+                NetworkManager.NetworkMetrics.TrackRpcSent(NetworkManager.ServerClientId, NetworkObjectId, rpcMethodName, (ulong)serializer.Writer.GetStream().Length);
+            }
```

Comment:
Are these the changes that are already in develop?

Reply:
What do you mean? There are no changes in develop

Reply:
Oh, oops. I misinterpreted what this was